src/components/api.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios"
+
+export function fetchArticleData() {
+    return axios
+    .get("https://be-nc-news-project.onrender.com/api/articles")
+}
+
+export function fetchArticleById(article_id) {
+    return axios
+    .get(`https://be-nc-news-project.onrender.com/api/articles/${article_id}`)
```

Comment:
I like that you've extracted these out into their own file. Take a look at axios.create to make this code a bit DRY-er!